tests/system/python/e2e/test_e2e_pi_scaleset.py

```diff
@@ -39,7 +39,7 @@
 SENSOR_VALUE = 21
 
 # scale(set) factor
-SCALE = "9/5"
+SCALE = "1.8"
```

Comment:
So, the problem occurs due to the scale-set filter now using a list of objects with a float type, whereas it was previously in JSON format.